src/main/scala/org/deeplearning4j/scalphagozero/board/GoBoard.scala

```diff
@@ -85,22 +76,22 @@ class GoBoard(val row: Int, val col: Int) {
           this.replaceString(neighborString.get.withLiberty(Point(point._1, point._2)))
         grid.remove(point)
       }
-      hash ^= ZobristHashing.ZOBRIST(((point._1, point._2), Some(goString.color))) //Remove filled point hash code.
-      hash ^= ZobristHashing.ZOBRIST(((point._1, point._2), None)) //Add empty point hash code.
+      hash ^= ZobristHashing.ZOBRIST((point._1, point._2, Some(goString.color))) //Remove filled point hash code.
+      hash ^= ZobristHashing.ZOBRIST((point._1, point._2, None)) //Add empty point hash code.
     }
 
   private def replaceString(newString: GoString): Unit =
     for (point <- newString.stones)
       grid += (point -> newString)
 
   def isSelfCapture(player: Player, point: Point): Boolean = {
-    val friendlyStrings: util.ArrayList[GoString] = new util.ArrayList[GoString]()
+    val friendlyStrings: ListBuffer[GoString] = ListBuffer.empty[GoString]
```

Comment:
ArrayList was no conscious decision against anything, just rushing. this is obviously better.